src/icons/file-icon/FileIcon.tsx

```diff
@@ -166,7 +166,7 @@ const EXTENSIONS: { [key: string]: { [key: string]: string } } = {
     IconFilePresentation: mirror(['odp', 'otp', 'pot', 'potx']),
     IconFilePowerpointPresentation: mirror(['ppt', 'pptx', 'pptm']),
     IconFilePhotoshop: mirror(['psd']),
-    IconFileSpreadsheet: mirror(['ods', 'xlt', 'xltx']),
+    IconFileSpreadsheet: mirror(['csv', 'tsv', 'ods', 'xlt', 'xltx']),
```

Comment:
nit: the extensions should be sorted
```suggestion
    IconFileSpreadsheet: mirror(['csv', 'ods', 'tsv', 'xlt', 'xltx']),
```